src/main/java/com/simplewebapp/service/UserService.java

```diff
@@ -9,14 +9,14 @@
  */
 public interface UserService {
 
-  public void addUser(User user);
+  void addUser(User user);
```

Comment:
Is it useful to returned the saved object. I believe Hibernate returns the saved object when persisted to the DB. This way you can have the saved object in memory in case you need it. The difference with the object you pass in as parameter is that ID is populated.